Sources/SwiftStylable/Classes/Style/Stylers/SpacingStylable.swift

```diff
@@ -8,5 +8,26 @@
 import UIKit
 
 protocol SpacingStylable : AnyObject {
-	var spacing: CGFloat { get set }
+	var spacing: CGFloat? { get set }
+    var imagePadding: CGFloat? { get set }
```

Comment:
Ook hier verschilt de leading spacing van de properties, zie comment hierboven

Reply:
Hier hetzelfde verhaal.
Ik zie al waar het "mis" gaat. In dit project zijn intents nog al "tab" gebruikt ipv 4 spaties. Dus die imagePadding heeft 4 spaties en de spacing heeft een "tab" ervoor. Dus daar komt het verscheel vandaan. Ga ik voor nu even niet aanpassen.